src/adapter/breakpoints/hitCondition.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { ProtocolError, invalidHitCondition } from '../../dap/errors';
+
+/**
+ * Regex used to match hit conditions. It matches the operator in group 1 and
+ * the constant in group 2.
+ */
+const hitConditionRe = /^(>|>=|={1,3}|<|<=|%)?\s*([0-9]+)$/;
```

Comment:
The one change I made here was allowing 1 to 3 `=`, since I type == or === by force of habit.